UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/TileChangeManager.cs

```diff
@@ -142,6 +142,12 @@ public LayerTile GetLayerTile(Vector3Int cellPosition, LayerType layerType)
 		return metaTileMap.GetTile(cellPosition, layerType);
 	}
 
+	[Server]
+	public MetaTileMap GetMetaTileMap()
+	{
+		return metaTileMap;
+	}
+
```

Comment:
it's not java, you can use
`public MetaTileMap MetaTileMap => metaTileMap;`
instead

Reply:
Thanks, I meant to ask about this, this was my bandaid solution.